images/users/hooroo.svg

```diff
@@ -0,0 +1 @@
,78.89 C305.715,87.4 301.463,102.292 288.694,102.292 C279.186,102.292 275.924,94.658 275.924,87.4 C275.924,77.765 281.308,64.619 292.569,64.115 C297.956,63.864 301.711,66.745 303.594,69.631 C304.842,71.626 305.715,74.507 305.715,78.89 M370.44,68.05 C370.44,78.946 364.984,98.014 348.638,98.014 C336.466,98.014 332.298,88.245 332.298,78.946 C332.298,66.603 339.189,49.78 353.613,49.142 C360.499,48.816 365.308,52.499 367.711,56.191 C369.319,58.76 370.44,62.441 370.44,68.05 M302.7,18.687 C303.325,15.891 300.317,15.283 299.306,15.194 C271.075,15.262 258.059,37.953 258.059,37.953 L259.411,31.593 C260.09,28.42 257.995,25.825 254.748,25.825 L241.798,25.825 C238.555,25.825 235.308,28.416 234.592,31.581 L216.617,111.174 C215.895,114.34 217.961,116.929 221.208,116.929 L234.319,116.929 C237.562,116.929 240.803,114.34 241.525,111.178 L241.692,110.451 C242.339,106.84 247.659,83.752 248.243,81.145 C252.921,59.924 266.981,41.424 291.043,39.651 C291.069,39.647 291.089,39.645 291.114,39.641 C297.027,38.696 298.783,35.508 299.445,33.407 L302.653,18.976 C302.667,18.883 302.673,18.79 302.7,18.687 M219.719,68.369 C219.719,48.499 206.903,29.746 179.655,29.746 C159.029,29.746 145.009,40.991 137.566,54.545 C132.669,51.085 126.227,48.972 118.157,48.972 C102.485,48.972 91.673,57.283 85.748,67.482 L86.826,62.894 C87.47,59.819 88.118,56.582 88.118,53.351 C88.118,38.96 76.961,31.837 64.99,31.837 C52.378,31.837 45.003,39.417 45.003,39.417 C42.741,41.742 41.473,41.057 42.193,37.891 L49.438,5.771 C50.158,2.6 48.087,0.008 44.841,0.008 L31.574,0.008 C28.331,0.008 25.083,2.596 24.363,5.76 L0.311,111.321 C-0.413,114.488 1.657,117.074 4.898,117.074 L18.169,117.074 C21.417,117.074 24.663,114.488 25.386,111.321 L34.748,70.33 C36.362,64.993 37.495,60.63 40.244,57.229 C42.992,53.674 47.193,50.928 51.726,50.928 C54.636,50.928 61.428,52.059 61.428,61.113 C61.428,63.699 60.659,67.014 60.659,67.014 C59.933,70.173 58.743,75.345 58.027,78.514 L49.574,115.262 C48.854,118.425 50.92,121.013 54.169,121.013 L67.277,121.013 C70.524,121.013 73.777,118.425 74.517,115.262 L80.52,89.722 C81.529,100.991 88.323,117.569 111.531,117.569 C124.016,117.569 133.696,112.197 140.167,104.487 C146.189,111.916 155.872,117.569 171.163,117.569 C202.092,117.569 219.719,91.928 219.719,68.369 M129.677,78.89 C129.677,87.4 125.422,102.292 112.654,102.292 C103.144,102.292 99.885,94.658 99.885,87.4 C99.885,77.765 105.268,64.615 116.538,64.118 C121.917,63.864 125.679,66.749 127.548,69.628 C128.803,71.631 129.677,74.506 129.677,78.89 M194.397,68.05 C194.397,78.946 188.949,98.014 172.602,98.014 C160.424,98.014 156.255,88.245 156.255,78.946 C156.255,66.603 163.146,49.78 177.571,49.142 C184.46,48.816 189.273,52.505 191.673,56.191 C193.276,58.76 194.397,62.441 194.397,68.05 M258.328,152.86 C281.221,167.118 308.992,171.911 335.419,169.037 C356.127,166.791 381.939,158.75 391.811,139.316 C394.832,133.365 391.008,114.977 387.06,118.568 C353.602,148.956 303.099,161.076 260.892,143.023 C257.782,141.691 254.4,150.42 258.328,152.86"/></svg>
```

Comment:
I looked at the SVG and for some reason there is lot of blank space on the right of the logo? Could you fix this?